tests/components/iron_os/conftest.py

```diff
@@ -107,6 +109,23 @@ def mock_ble_device() -> Generator[MagicMock]:
         yield ble_device
 
 
+@pytest.fixture(autouse=True)
+async def mock_github(
+    aioclient_mock: AiohttpClientMocker,
+) -> AiohttpClientMocker:
+    """Mock aiogithubapi."""
+
+    aioclient_mock.get(
+        "https://api.github.com/repos/Ralim/IronOS/releases/latest",
+        json={
+            **json.loads(load_fixture("releases_latest.json", DOMAIN)),
+        },
+        headers=json.loads(load_fixture("base_headers.json", DOMAIN)),
+    )
+
+    return aioclient_mock
```

Comment:
Instead of mocking the aioclient, shouldn't we mock the github library?

Reply:
I tried but i couldnt't get it to work. The github integration uses the same library and does the same, it mocks the aioclient.